zppy/templates/coupled_global.py

```diff
@@ -112,28 +113,80 @@ def add_trend(
 # -----------------------------------------------------------------------------
 # Function to get ylim
 def get_ylim(standard_range, extreme_values):
-    standard_min = standard_range[0]
-    standard_max = standard_range[1]
-    if extreme_values == []:
-        return [standard_min, standard_max]
-    extreme_min = np.amin(extreme_values)
-    extreme_max = np.amax(extreme_values)
-    if standard_min <= extreme_min:
-        ylim_min = standard_min
+    if len(extreme_values) > 0:
```

Comment:
I updated this function to use a better algorithm for determining the `y` range.